modular_skyrat/modules/aesthetics/lights/code/lighting.dm

```diff
@@ -10,13 +10,13 @@
 	overlay_icon = 'modular_skyrat/modules/aesthetics/lights/icons/lighting_overlay.dmi'
 	brightness = 6.5
 	fire_brightness = 4.5
-	fire_colour = "#D47F9B"
-	bulb_colour = LIGHT_COLOR_FAINT_BLUE
+	fire_colour = COLOR_FIRE_LIGHT_RED
+	bulb_colour = COLOR_OFF_WHITE
 	bulb_power = 1.15
 	nightshift_light_color = null // Let the dynamic night shift color code handle this.
-	bulb_low_power_colour = LIGHT_COLOR_DARK_BLUE
-	bulb_low_power_brightness_mul = 0.4
-	bulb_low_power_pow_min = 0.4
+	bulb_low_power_colour = LIGHT_COLOR_BROWN
```

Comment:
Would be nice to actually see what those look like now, there doesn't seem to be much in terms of low power lights in your examples. I'm not certain that brown will necessarily look the best either.

Reply:
Low power mode is when there is no power, running on the backup cell. Going for a dim emergency light look. Warm temp, some red, but not as red as the fire alarm/emergency lighting.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/7732245d-3025-40f0-a451-f8ab54377659)

Reply:
Current low power:

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/fa141f7f-8d44-4c61-a7e1-f88fbc4381c0)

Reply:
Fire alarm after:

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/ee7ccaee-2856-4735-a822-b38ced4b0370)

Fire alarm current:

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/f45c93d5-639f-4cf1-880e-920518882406)